tribits/python_utils/gitdist.py

```diff
@@ -1312,7 +1312,10 @@ def getCommandlineOps():
 
   clp.add_option(
     noColorArgName, dest="useColor", action="store_false",
-    help="If set, don't use color in the output for gitdist (better for output to a file).",
+    help="If set, don't use color in the output for gitdist and set"
+    +" '-c color.status=never' before the git command (like 'status')."
+    +"  NOTE: user should also pass in --color=never for git commands "
+    +" accept that argument.  (Better for output to a file).",
```

Comment:
```
for git commands that accept that argument.
                 ^^^^ missing?
```

Reply:
I wanted to make the script as future proof as possible so not make it know too much about Git.  Currently, gitdist only knows a few Git commands in order to provide repo states and do a few other things.